patches/@chialab+estransform+0.17.3.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/node_modules/@chialab/estransform/dist/index.js b/node_modules/@chialab/estransform/dist/index.js
+index 48e09e4..fea244a 100644
+--- a/node_modules/@chialab/estransform/dist/index.js
++++ b/node_modules/@chialab/estransform/dist/index.js
+@@ -1,10 +1,6 @@
+ import { dirname as __pathDirname } from 'path';
+ import { createRequire as __moduleCreateRequire } from 'module';
+ 
+-const require = __moduleCreateRequire(import.meta.url);
+-const __filename = new URL(import.meta.url).pathname;
+-const __dirname = __pathDirname(__filename);
```

Comment:
these cause Syntax error, probably "@chialab" modules are not compiled correctly, need to check with them

Reply:
did we create issues for these patches or reach out to chialab?

Reply:
I filed a bug here https://github.com/chialab/rna/issues/150

Reply:
the bug was fixed